extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,7 +155,52 @@ export class ConsentConfig {
       }
     }
 
-    return config;
+    return this.mergeGeoOverride_(config).then(() => config);
+  }
+
+  /**
+   * Merge correct geoOverride object into toplevel config, then
+   * validate.
+   * @param {JsonObject} config
+   * @return {!Promise}
+   */
+  mergeGeoOverride_(config) {
+    if (config['geoOverride']) {
+      userAssert(
+        config['geoOverride']['geoGroupUnknown'],
+        'geoGroupUnknown must be specified in geoOverride'
+      );
+      Services.geoForDocOrNull(this.element_).then(geoService => {
+        userAssert(geoService, 'requires <amp-geo> to use geoOverride');
+        if (geoService.ISOCountry === 'unknown') {
```

Comment:
as mentioned above, geoGroupUnknown is not our reserved keyword to handle `unknown`.

Reply:
removed